### DISCUSSION TITLE
Using Enum Array on Model

### DISCUSSION THREAD

**Discussion by progressivetr** [TypedProject/ts-express-decorators#1426] [Q&A] [answered]
Hello.

Which decoration should I use for defining enum array in json model?

**Comment by Romakita** [comment, marked as answer]
Just @Enum()

```typescript
class MyModel {
   @Enum(MyEnum)
   enums: MyEnum[];
}
```

See you
Romain

**Reply by Romakita** [reply]
@progressivetr Can you close this discussion please ?

**Reply by progressivetr** [reply]
It works. Thanks

**Reply by progressivetr** [reply]
How do i close discussion?

**Reply by Romakita** [reply]
Just mark the discussion as answered
